### PR TITLE
Updated recetox tools which require docker containers

### PR DESCRIPTION
**Pull request by hechth**
Added some configurations for recetox tools which are only running in docker.

### CODE REVIEW

File: tools.yml [hunk 1]
```diff
@@ -2416,8 +2416,32 @@ tools:
...
+    cores: 4
+    mem: 32
   toolshed.g2.bx.psu.edu/repos/recetox/recetox_aplcms_align_features/recetox_aplcms_align_features/.*:
     cores: 4
+  toolshed.g2.bx.psu.edu/repos/recetox/recetox_aplcms_generate_feature_table/recetox_aplcms_generate_feature_table/.*:
+    mem: 16
+  toolshed.g2.bx.psu.edu/repos/recetox/recetox_aplcms_recover_weaker_signals/recetox_aplcms_recover_weaker_signals/.*:
+    mem: 16
+  toolshed.g2.bx.psu.edu/repos/recetox/recetox_msfinder/recetox_msfinder/.*:
+    mem: 64
```
Comment (bgruening) [inline tools.yml:2441]: That is a lot! Does this always needs 64 or can we add a rule to take less for smaller files?

Comment (hechth) [reply tools.yml:2441]: We can also take it down to 32GB - so far we haven't had major issues. Internally we run the tool on Windows Server VMs with 256GB RAM.

Comment (bgruening) [reply tools.yml:2441]: As less as possible I would say :)

Comment (hechth) [reply tools.yml:2441]: Then we can try 16GB and see if we run into errors.

**Comment by hechth** [comment]
@bgruening can this be merged?

**Review by bgruening — APPROVED** [review]

**Pull request merged**